app/components/Modal/index.js

```diff
@@ -1,76 +1,59 @@
 // @flow
 
-import type { Node } from 'react';
-
-import { Component } from 'react';
 import { Modal as ReactModal } from 'react-overlays';
 import cx from 'classnames';
 import Icon from 'app/components/Icon';
 import styles from './Modal.css';
 
 type Props = {
-  children: Node,
-  onHide: () => any,
-  closeOnBackdropClick: boolean,
   show: boolean,
-  keyboard: boolean,
-  onKeyDown: (e: KeyboardEvent) => void,
+  children: React$Node,
```

Comment:
Any special reason for changing the type here? I might be missing something

Reply:
`React$Node` is an alias for the `Node`-type we previously imported from React. It just saves us an import

Reply:
Right, so only to remove an import - gotcha!